assets/style.css

```diff
@@ -19,6 +19,15 @@ body {
   font-family: 'Poppins', arial, sans-serif;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
+/* FONTS */
+h1{
+  font-family: 'Domine', serif;
+}
```

Comment:
@ZubairImtiaz3 why did you add the fonts?

Reply:
The font target the main heading which is Contribute to this project. This was added long ago by me n that pull request was merged.
I didn't add it again but dunno why it is added.

I think there i might not be using the version control system correctly

Reply:
@ZubairImtiaz3, kindly remove those lines that are unnecessary...

Reply:
Ok